src/ddmd/hdrgen.d

```diff
@@ -2975,7 +2975,7 @@ public:
         }
     }
 
-    void parametersToBuffer(Parameters* parameters, int varargs)
+    void parametersToBuffer(Parameters* parameters, int varargs, Expressions* fargs = null)
```

Comment:
I know it's on purpose.
This has to go into a separate function.
maybe `parameterErrorsToBuffer` ?